### ISSUE TITLE
Release SUPERB finetuned checkpoints

### ISSUE THREAD

**Issue by ashishBT** [s3prl/s3prl#151]
I am using HuBERT Base for ASR task on LibriSpeech dataset, but unable to reproduce the results mentioned in the [article](https://arxiv.org/abs/2105.01051) . I am using following command to generate test results on LibriSpeech test-clean.

`python3 run_downstream.py -m evaluate -t "test-clean" -n ExpName -u hubert_base -d asr`

**Output:**

```
/content/drive/My Drive/Colab Notebooks/SUPERB/demoEnv/lib/python3.7/site-packages/torchaudio/backend/utils.py:54: UserWarning: "sox" backend is being deprecated. The default backend will be changed to "sox_io" backend in 0.8.0 and "sox" backend will be removed in 0.9.0. Please migrate to "sox_io" backend. Please refer to https://github.com/pytorch/audio/issues/903 for the detail.
  '"sox" backend is being deprecated. '
[Runner] - Start a new experiment
--2021-07-16 06:05:21--  https://dl.fbaipublicfiles.com/hubert/hubert_base_ls960.pt
Resolving dl.fbaipublicfiles.com (dl.fbaipublicfiles.com)... 104.22.75.142, 172.67.9.4, 104.22.74.142, ...
Connecting to dl.fbaipublicfiles.com (dl.fbaipublicfiles.com)|104.22.75.142|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1136468879 (1.1G) [application/zip]
Saving to: ‘/root/.cache/torch/hub/s3prl_cache/5e6b91abd59b390dc3f89225f0e7d26f5bcb6fac496a08110c8862e3b1bb93e7’

/root/.cache/torch/ 100%[===================>]   1.06G  68.2MB/s    in 16s     

2021-07-16 06:05:37 (69.4 MB/s) - ‘/root/.cache/torch/hub/s3prl_cache/5e6b91abd59b390dc3f89225f0e7d26f5bcb6fac496a08110c8862e3b1bb93e7’ saved [1136468879/1136468879]

[Featurizer for <class 'upstream.hubert.expert.UpstreamExpert'>] - The input upstream is only for initialization and not saved in this nn.Module
[Featurizer for <class 'upstream.hubert.expert.UpstreamExpert'>] - Take a list of 13 features and weighted sum them.
ASR dataset train: 100% 28551/28551 [00:10<00:00, 2618.82it/s]
ASR dataset test-clean: 100% 2620/2620 [00:00<00:00, 40892.63it/s]
test-clean: 100% 2620/2620 [07:07<00:00,  6.14it/s]
test-clean loss: 9.304394721984863
test-clean uer: 138.35627177106554
test-clean wer: 2006.7175572519084
```

Could you please correct me if I made any mistakes.

**Comment by leo19941227** [comment]
Hi,

This is because you did not assign which checkpoint to test in the evaluation mode.
Please refer to [here](https://github.com/s3prl/s3prl/tree/master/s3prl/downstream#testing-without-lm) for the correct usage.

**Comment by ashishBT** [comment]
Hi,
Thanks for your quick reply.
As per my understanding, to create these checkpoints, I need to train the model on full training LibriSpeech data.
Is there any pretained checkpoint available to use directly?

**Comment by leo19941227** [comment]
Yes you are correct and yes there pretrained checkpoints. But we are still working on releasing them.
Some of them are trained with Facebook's infra and will require additional procedure to release them.
I will rename this issue and close it after we release our finetuned checkpoints. (in one or two weeks.)

**Comment by andi611** [comment]
> Hi,
> As per my understanding, to create these checkpoints, I need to train the model on full training LibriSpeech data.

Note that our fine-tuned checkpoints are trained with 100 hours of LibriSpeech, and not the full amount of 960 hours.
As a result, it is not computational expensive, and you are most likely possible to fine-tune them by yourself (the training time still depends on your machine, but it should be affordable).

**Comment by leo19941227** [comment]
The issue is resolved by [the latest doc update which releases the training artifacts](https://github.com/s3prl/s3prl/blob/master/s3prl/downstream/docs/superb_artifacts.md)!
